app/models/setting/provisioning.rb

```diff
@@ -101,4 +107,8 @@ def self.map_pxe_kind
       yield pxe_kind, templates
     end
   end
+
+  def self.default_excluded_facts(ignored_interfaces = IGNORED_INTERFACES)
+    ignored_interfaces
+  end
```

Comment:
Can you optionally add couple of tests to test IGNORED_INTERFACES constant actually? We are not sure we ship with default value that just works. A single test with different names "Ipaddress_vnet1" should do it "Ipaddress:vnet1".